packages/app/src/systems/Asset/hooks/useAssetForm.tsx

```diff
@@ -11,7 +11,7 @@ import type { Maybe } from '~/systems/Core';
 export type AssetFormValues = {
   name: string;
   assetId: string;
-  imageUrl: string;
+  imageUrl: string | undefined;
```

Comment:
```suggestion
  imageUrl?: string;
```

Reply:
on this way it breaks yupResolver typing
![image](https://github.com/FuelLabs/fuels-wallet/assets/8636507/ca24ecb0-a33e-4306-af2a-cb68d20165e1)